### ISSUE TITLE
request

### ISSUE THREAD

**Issue by NguefackLoic** [iqbalmbadhan/first-django-blog#1]
Hi sorry for the inconvenience I would like to know if you have solved your problem on the wordpress theme HUMAN CONSULT?

**Comment by NguefackLoic** [comment]
Hi sorry for the inconvenience I would like to know if you have solved your problem on the HUMAN CONSULT wordpress theme?

**Comment by iqbalmbadhan** [comment]
No, it was for another template called Red Art and the issue isn't resolved yet.
